Sources/Temporal/Client/TemporalClient.swift

```diff
@@ -154,21 +154,19 @@ public final class TemporalClient: Sendable {
     /// - Parameters:
     ///   - transport: The transport layer that should be used by the gRPC connection of the ``TemporalClient``.
     ///   - configuration: The configuration of the ``TemporalClient``.
-    ///   - isolation: The isolation domain of the caller.
     ///   - logger: The logger used in the ``TemporalClient``.
     ///   - body: A closure which is called with the ``TemporalClient``. When the closure returns, the ``TemporalClient`` is shutdown gracefully.
     /// - Returns: The result of the passed closure.
     public static func connect<Transport: ClientTransport, Result: Sendable>(
         transport: Transport,
         configuration: TemporalClient.Configuration,
-        isolation: isolated (any Actor)? = #isolation,
```

Comment:
I guess with nonisolated nonsending it becomes discouraged to pass in the isolation manually?